spec/services/step_by_step_filter/options_spec.rb

```diff
@@ -2,69 +2,17 @@
 
 RSpec.describe StepByStepFilter::Options do
   it "returns filter options with the default selected" do
-    subject = described_class.statuses
-
-    all_option = {
-      text: "All",
-      data_attributes: {
-        show: "all",
-      },
-      selected: true,
-    }
-
-    expect(subject).to include(all_option)
-
-    StepByStepPage::STATUSES.each do |status|
-      status_option = {
-        text: status.humanize,
-        value: status,
-        data_attributes: {
-          show: status,
-        },
-        selected: false,
-      }
-
-      expect(subject).to include(status_option)
-    end
+    expect(described_class.statuses)
+      .to include(text: "All",
+                  data_attributes: { show: "all" },
+                  selected: true)
```

Comment:
This test no longer does quite what it says on line 4 - it only checks if an `All` option exists.

I'd suggest checking that `described_class.statuses.count` is equal to  something like `step_by_step_page.STATUSES.count + 1`

Reply:
Ah ha did I go too far? I've added an extra test in.